docs/configuration/README.md

```diff
@@ -37,8 +37,8 @@ var chartDifferentHoverMode = new Chart(ctx, {
 Options may be configured directly on the dataset. The dataset options can be changed at 3 different levels and are evaluated with the following priority:
 
 - per dataset: dataset.*
-- per chart: options.datasets[type].*
-- or globally: Chart.defaults.datasets[type].*
+- per chart: options.[type].datasets*
+- or globally: Chart.defaults[type].datasets.*
```

Comment:
`Chart.defaults.[type].datasets.*`

Reply:
This was written in JS style. I.e. an array access. I was trying to keep it in line with how it was before

Reply:
Right. It should actually be `Chart.defaults.<type>.datasets.*`, because the type is not optional. But its going to get copied to code so probably better in JS style. Maybe even add the `var type='myChartType'` above 🙊